### ISSUE TITLE
The win2016-standard templates use a template variable to define the "memory" user variable

### ISSUE THREAD

**Issue by arizvisa** [boxcutter/windows#198]
List generated by the following...
```
$ packer --version
1.5.1
$ for name in *.json; do packer validate $name 1>/dev/null || printf '- [ ] %s\n' "$name"; done
```

- [x] win2012r2-standard.json
- [x] win2016-standard-cygwin.json
- [x] win2016-standard-ssh.json
- [ ] win7x64-enterprise-ssh.json
- [ ] win7x64-pro-cygwin.json
- [ ] win7x64-pro-ssh.json
- [ ] win7x86-enterprise-cygwin.json
- [ ] win7x86-enterprise-ssh.json
- [ ] win7x86-pro-cygwin.json
- [ ] win7x86-pro-ssh.json
- [ ] win81x64-enterprise-cygwin.json
- [ ] win81x64-enterprise-ssh.json
- [ ] win81x64-pro-cygwin.json
- [ ] win81x64-pro-ssh.json
- [ ] win81x86-pro-cygwin.json

**Comment by arizvisa** [comment]
This issue is related to #196, and issue #194. I think this issue covers the rest of the templates that have a bad/outdated schema.

**Comment by arizvisa** [comment]
This is closed by PR #199.
